components/NavBar.js

```diff
@@ -1,5 +1,71 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import styles from '../styles/Navbar.module.scss'
+
+import { useAppContext } from '../context/app-context'
+
 export default function NavBar() {
+  const { sharedState } = useAppContext()
+
+  const courses = sharedState.courses.map((course, i) => {
+    const modules = course.Modules.map((mod, j) => {
+      return (
+        <Link key={j} href='/moduleDashboard'>
+          <a>{mod.name}</a>
+        </Link>
+      )
+    })
+
     return (
-        <div></div>
+      <div key={i} className={styles.course}>
+        <Link href='/courseDashboard'>
+          <a>{course.Name}</a>
+        </Link>
+        <div className={styles.content}>
+          <Link href='/moduleDashboard'>
+            <a>All Modules</a>
+          </Link>
+
+          {modules}
+
+          <Link href='/addModuleForm'>
+            <a>+ Add New Module</a>
+          </Link>
```

Comment:
Yeah I think this is a good idea in the future! We could so something like 'courseDashboard/:courseID'. I'll make an issue for this so that we can implement it in a later PR.